move-mutation-test/src/lib.rs

```diff
@@ -68,32 +65,13 @@ pub fn run_mutation_test(
     benchmarks.total_duration.start();
 
     // Run original tests to ensure the original tests are working:
-
-    // We need to check for the latest git deps only for the first time we run the test.
-    // All subsequent runs with this tool will then have the latest deps fetched.
-    let skip_fetch_deps = false;
-    // TODO: use this one instead once it is available in the aptos-core.
-    //let skip_fetch_deps = test_config.move_pkg.skip_fetch_latest_git_deps; // false by default
-    let result = run_tests(
-        test_config,
-        &package_path,
-        skip_fetch_deps,
-        &mut error_writer,
-    );
-
-    if let Err(e) = result {
-        let msg =
-            format!("Test suit is failing for the original code! Unit test failed with error: {e}");
-        error!("{msg}");
-        return Err(anyhow!(msg));
-    }
+    run_tests_on_original_code(test_config, &package_path)?;
```

Comment:
I decided to make two separate functions where one can find the differences more easily.
It makes our code cleaner here